### PR TITLE
#1267 Missing StocktakesPage toggle

### LINKED ISSUE #1267: Missing StocktakesPage toggle
## Describe the bug

Missing Old/Current toggle on `StocktakesPage`

### To reproduce
Go to page no see toggle

### Expected behaviour

The toggle should be there

### Version and device info

2.3.6 development

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #1267 

## Change summary

- Adds back the toggle I forgot about

## Testing

See #1043 

### Related areas to think about

Conflicts with #1266 

Note that this messes up the back navigation a little bit. When navigating back `refreshData` is called which will just display everything. This needs to change with changes made in #1266 such that the state is re-init, rather than `refreshData`

### CODE REVIEW

File: src/pages/dataTableUtilities/actions/tableActions.js [hunk 1]
```diff
@@ -62,6 +62,26 @@ export const hideOverStocked = () => ({ type: ACTIONS.HIDE_OVER_STOCKED });
  */
 export const hideStockOut = () => ({ type: ACTIONS.HIDE_STOCK_OUT });
 
+/**
+ * Hides all rows which have a status of finalised.
+ */
+export const showFinalised = () => ({ type: ACTIONS.SHOW_FINALISED });
```
Comment (mark-prins) [inline src/pages/dataTableUtilities/actions/tableActions.js]: :nit this one `showFinalised` and that one `showNotFinalised` are only called by an internal method, so don't need to be exported. There's a really minor overhead in the export, but it also declares intention - and I wondered if those guys aren't supposed to be called externally?

Comment (joshxg) [reply src/pages/dataTableUtilities/actions/tableActions.js]: I wonder this myself - I landed on "ahhh, why not" .. not sure what the best thing is. No harm in being called directly, but the intention is that the wrapper is called, for ease of use

Comment (joshxg) [reply src/pages/dataTableUtilities/actions/tableActions.js]: oh - I also hate these names if you have any other suggestions 🤣 - I didn't want to use `past` and `current`, I felt they were less explicit ?

Comment (mark-prins) [reply src/pages/dataTableUtilities/actions/tableActions.js]: might as well leave them then - if ok to call directly.

I stumbled on the Past and Current names later on (as in, didn't know what they meant at first) - so prefer the `showFinalised` and `showNotFinalised` My only other suggestion would make the naming worse, so let's not go there ( including 'only' to make it clear what is being shown `showOnlyFinalised` or `showFinalisedOnly` - but I like that idea less than what you have ). also, elsewhere in mSupply we have unfinalised rather than not. but not is more logical.

so, in summary. all good as it is. thanks.
